<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Script</class>
 <widget class="QDialog" name="Script">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbScript">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Script</string>
   </property>
   <widget class="QPushButton" name="pbLoadScript">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>211</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select script file and play</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leScriptFile">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>50</y>
      <width>206</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="pteScriptStatus">
    <property name="geometry">
     <rect>
      <x>85</x>
      <y>80</y>
      <width>206</width>
      <height>91</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAbort">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>100</y>
      <width>76</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
